CONTRIBUTING.md

```diff
@@ -118,9 +118,9 @@ Whether you edit any tutorial or not doesn't matter.
 
 To avoid having to run the tutorials, you may download executed tutorial artifacts named "tutorial" from our CI (see the capture below) and put them in `docs/build` before
 extracting the files in the zip to `docs/source/tutorial` directory.
-Note that the CI runs with Python 3.8 and the generated artifacts contain pickle files.
+Note that the CI runs with Python 3.12 and the generated artifacts contain pickle files.
 The pickle files are serialized with [the protocol version 5](https://docs.python.org/3/library/pickle.html#data-stream-format) so you will see the error with Python 3.7 or older.
-Please use Python 3.8 or later if you build the documentation with artifacts.
+Please use Python 3.12 or later if you build the documentation with artifacts.
```

Comment:
nit: According to the documentation, the Pickle protocol version has not been incremented since Python 3.8, so it may be fine to keep it as is.

Reply:
Thanks. Absolutely right. Alternatively, how about removing the previous line 

> The pickle files are serialized with [the protocol version 5](https://docs.python.org/3/library/pickle.html#data-stream-format) so you will see the error with Python 3.7 or older.
? 
Since a different python version from Optuna sphinx CI causes different behaviours of sphinx build when a contributor builds the docs locally.

Reply:
Agree. We should simply recommend building with the same Python version.